tests/test_hand.py

```diff
@@ -27,15 +27,20 @@ def six() -> TestCard:
 
 
 @pytest.fixture
-def ace_hand(ace: TestCard, ten: TestCard) -> Hand:
+def ace_test_hand(ace: TestCard, ten: TestCard) -> Hand:
     return Hand(ace, ten)
 
 
 @pytest.fixture
-def sixteen_hand(six: TestCard, ten: TestCard) -> Hand:
+def sixteen_test_hand(six: TestCard, ten: TestCard) -> Hand:
     return Hand(six, ten)
 
 
+def test_hand(*card_values: int) -> Hand:
+    TestCards = [TestCard(card_value) for card_value in card_values]
+    return Hand(*TestCards)
+
+
 def test_starting_hand_value(ace_hand: Hand, sixteen_hand: Hand) -> None:

```

Comment:
Quite a lot of different assertions going on in one test. Really, tests should be testing only one thing at a time.

Reply:
Thank you, I have split the test_starting_hand_value into two seperate test cases